optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -276,12 +277,12 @@ def __call__(self, trial):
             self.best_booster = booster
 
         if self.pbar is not None:
-            self.pbar.set_description(pbar_fmt.format(self.action, self.best_score))
+            self.pbar.set_description(pbar_fmt.format(self.step_name, self.best_score))
             self.pbar.update(1)
 
         self.report.append(
             dict(
-                action=self.action,
+                action=self.step_name,
```

Comment:
How about leaving a comment that the key name differs from a historical reason?

Reply:
I add it. Thanks. Please note that I plan to remove the `self.report` and use `study` to retrieve the results in the future PRs, and then the comment will be removed too.
https://github.com/optuna/optuna/pull/1032/commits/ef51176e701105e99c6068f6d28daeb6dfc5d953#diff-0d8b1d930818af2b13939666bb2df3eaR285-R286